mandatory/2-function-creation.js

```diff
@@ -16,17 +25,28 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(num) {}
-
+function formatPercentage(num) {
+let newNum=Math.round(num*100)/100;
+if (newNum > 100) newNum=100;
+return newNum+"%";
+}
 /*
 Write a function that:
 - takes an array of strings as input
 - removes any spaces in the beginning or end of each string
 - removes any forward slashes (/) in each string
 - makes all strings all lowercase
 */
-function tidyUpStrings(arrayOfStrings) {}
-
+function tidyUpStrings(arrayOfStrings) {
+  for (let i=0; i<arrayOfStrings.length; i++){
+    arrayOfStrings[i]= arrayOfStrings[i].trimStart();
```

Comment:
It's not wrong but i suggest using .trim() function would trim both start and end side of string, and will save you from writing one more line of code  :)

Reply:
Azin is right

Reply:
Instead of accessing and then overwriting the `arrayOfStrings[i]` value on each line, can you think of a way to simplify this?